src/main/java/fr/inria/diversify/mutant/pit/GradlePitTaskAndOptions.java

```diff
@@ -0,0 +1,58 @@
+package fr.inria.diversify.mutant.pit;
+
+/**
+ * Created by Daniele Gagliardi
+ * daniele.gagliardi@eng.it
+ * on 21/07/17.
+ */
+
+public class GradlePitTaskAndOptions {
+
+    public static boolean descartesMode = false;
+
+    public static boolean evosuiteMode = false;
+
+    public static final String OPT_WITH_HISTORY = "enableDefaultIncrementalAnalysis = ";
+
+    public static final String OPT_VALUE_REPORT_DIR = "reportDir = 'build/pit-reports'";
+
+    public static final String OPT_TARGET_CLASSES = "targetClasses = ";
+
+    public static final String OPT_VALUE_FORMAT = "outputFormats = ['CSV','HTML']";
+
+    public static final String OPT_TARGET_TESTS = "targetTests = ";
+
+    public static final String PROPERTY_ADDITIONAL_CP_ELEMENTS = "additionalMutableCodePaths = ";
+
+    public static final String PROPERTY_EXCLUDED_CLASSES = "excludedClasses = ";
+
+    //Apparently Unsupported by PIT plugin
+    public static final String OPT_ADDITIONAL_CP_ELEMENTS = "classPath = ";
```

Comment:
Yes, I know, but the problem is the following: if I add timeout paramters in the Gradle build file:
`pitest {`
`    targetClasses = ['example.*']`
`    enableDefaultIncrementalAnalysis = true`
`    reportDir = 'build/pit-reports'`
`    outputFormats = ['CSV','HTML']`

`    mutationEngine = 'descartes'`
`
    mutators = ['\'a\'']`
`    timeoutConst = 10000`
`}`
I get following error:
`Could not set unknown property 'timeoutConst' for object of type info.solidsoft.gradle.pitest.PitestPluginExtension.`  

If I add jvmArg parameter:
`pitest {`
`    targetClasses = ['example.*']`
`    enableDefaultIncrementalAnalysis = true`
`    reportDir = 'build/pit-reports'`
`    outputFormats = ['CSV','HTML']`

`    mutationEngine = 'descartes'`
`
    mutators = ['\'a\'']`
`    jvmArgs = ['8 GB']`
`}`
 the execution seems never finish:

`14.59.14 PIT >> INFO : MINION : Errore: impossibile trovare o caricare la classe principale 8G`
``
`<===========--> 85% EXECUTING [8m 35s]`
`> :pitest`
The same holds if I specify the value:
`    jvmArgs = ['16 GB']`